geniza/corpus/templates/corpus/snippets/document_shelfmarks.html

```diff
@@ -0,0 +1,10 @@
+{% for block in document.textblock_set.all %}
+    {% if block.certain %}
+        <span>{{ block.fragment.shelfmark }}</span>
```

Comment:
I think we need to add the ` + ` in the template rather than with css — when I tested locally and tried to cut and paste the shelfmark, it didn't include the ` + `; I suppose that will require extra logic to work with uncertain shelfmarks, though ...

Reply:
Done in e14654b 👍